src/Util/Strings.php

```diff
@@ -62,19 +63,19 @@ public static function escapeTags($string)
      * @brief Use this on "body" or "content" input where angle chars shouldn't be removed,
      * and allow them to be safely displayed.
      * @param string $string
-     * 
+     *
      * @return string
      */
     public static function escapeHtml($string)
     {
-        return htmlspecialchars($string, ENT_COMPAT, 'UTF-8', false);
+        return htmlentities($string, ENT_QUOTES | ENT_HTML5, "UTF-8", false);
```

Comment:
Code standard: Please keep the quoting style consistent with the rest of the file, which is single quotes for strings.

Reply:
Do you want me to revert the changes I made to `escapeHtml`? Or edit the unit test to comply with my changes?

Reply:
Yes please, and I've seen you did, thanks!